test/mavsdk_tests/mavsdk_test_runner.py

```diff
@@ -128,9 +128,11 @@ def __init__(self, model, workspace_dir, log_dir, speed_factor):
                     workspace_dir + "/build/px4_sitl_default/build_gazebo",
                     "GAZEBO_MODEL_PATH":
                     workspace_dir + "/Tools/sitl_gazebo/models",
-                    "PX4_SIM_SPEED_FACTOR": str(speed_factor)}
+                    "PX4_SIM_SPEED_FACTOR": str(speed_factor),
+                    "DISPLAY": os.environ['DISPLAY']}
         self.cmd = "gzserver"
-        self.args = [workspace_dir + "/Tools/sitl_gazebo/worlds/" +
+        self.args = ["--verbose",
```

Comment:
Actually, do we need this by default? It generates a lot of distracting (interleaved output) noise.

Reply:
For now, yes. but I'll cut it down and re-route all this output to logfiles later. It's already prepared.